README.md

```diff
@@ -251,18 +255,18 @@ Note that the Rust crate implementing the Python bindings is called `py-polars`
 Rust crate `polars` itself. However, both the Python package and the Python module are named `polars`, so you
 can `pip install polars` and `import polars`.
 
-## Use custom Rust function in Python?
+## Using custom Rust functions in Python
 
 Extending Polars with UDFs compiled in Rust is easy. We expose PyO3 extensions for `DataFrame` and `Series`
 data structures. See more in https://github.com/pola-rs/pyo3-polars.
 
 ## Going big...
 
-Do you expect more than 2^32 (~4.2 billion) rows? Compile Polars with the `bigidx` feature flag.
-
-Or for Python users, install `pip install polars-u64-idx`.
+Do you expect more than 2^32 (~4.2 billion) rows? Compile Polars with the `bigidx` feature
+flag or, for Python users, install `pip install polars-u64-idx`.
 
-Don't use this unless you hit the row boundary, as the default build of Polars is faster and consumes less memory.
+(Don't use this unless you hit the row boundary; the default build of Polars is faster
```

Comment:
Previous version was better.

Reply:
Not grammatically - should either omit the comma or use a semicolon ;)

Reply:
Fair enough, the comma should go. But in regular text, we shouldn't put full sentences in parentheses like this (that is what triggered me). I'll push an update.